qfieldsync/gui/layers_config_widget.py

```diff
@@ -128,7 +141,25 @@ def reloadProject(self):
 
         self.layersTable.setRowCount(0)
         self.layersTable.setSortingEnabled(False)
+
+        # Get filtered layers
+        show_visible_only = self.showVisibleLayersOnlyCheckbox.isChecked()
+        text_filter = self.textFilterBox.text().lower()
+
         for layer_source in self.layer_sources:
+            layer_name = layer_source.layer.name().lower()
+            layer_visible = (
+                QgsProject.instance()
+                .layerTreeRoot()
+                .findLayer(layer_source.layer.id())
+                .isVisible()
+            )
+            # Apply filter
+            if show_visible_only and not layer_visible:
+                continue
+            if (text_filter) and (text_filter not in layer_name):
+                continue
```

Comment:
```suggestion

            if text_filter and text_filter not in layer_name:
                continue
```
A line of air before and after keywords and keyword blocks is always welcome.
Also, note the braces are not required. I am actually surprise that `black` did not remove them.

Reply:
Great, noted!